Sources/SpeziAccessGuard/AccessGuardViewModel.swift

```diff
@@ -116,7 +117,8 @@ final class AccessGuardViewModel {
             locked = false
         }
     }
-    
+
+    @MainActor
```

Comment:
This already runs on the Main Thread. There is no need to do annotate this function.